### ISSUE TITLE
[QUESTION]: Re-training or New Training?

### ISSUE THREAD

**Issue by spyda90** [chemprop/chemprop#799]
Hi,

I've built a hERG prediction model that seems to perform quite well.

However, in the development of a totally new chemical class, it is not so good in predicting hERG inhibition, probably due to the lack of information in that chemical space.

I am thinking to add those in-house data related to that chemical class (or all our in-house data) to train the model.

I have a couple of questions on that:
1) Does it make sense? We are talking about 15 data points for the chemical class of interest or 80 data points in case of the whole in-house set, while the original training set is >10k data points...
2) In general, which is the best way to go? Is there a method to re-train the model generated with Chemprop using an additional dataset or I have to simply add those data points to the original training set and re-train it from the beginning (including hyperparameter optimization)?

Thank you in advance for your help.

Francesco

**Comment by kevingreenman** [comment]
Hi @spyda90, we did active learning with chemprop in [this paper](https://www.science.org/doi/10.1126/science.adi1407) and found that adding just a small number of datapoints to a relatively large set can help a lot if what you're adding is similar to the space where you care about having good predictions. We retrained the models from scratch (without HPO), but this isn't necessarily the only way to do it. You can certainly try the [pretraining / transfer learning strategy](https://github.com/chemprop/chemprop?tab=readme-ov-file#pretraining).

**Comment by spyda90** [comment]
Hi Kevin,

Thank you for your answer!
I have tried retraining the models from scratch (without HPO). The performance in the validation set are mantained.

I am going to test it in comparison with the older ones as sson as I have new generated data available.

I will keep you posted.

Ciao,
Francesco
